src/index.html

```diff
@@ -15,8 +15,90 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="https://snipboard.io/">
+          <img
+            src="images/logo.png"
+            alt="logo Moyo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav_link is-active"
+              href="https://snipboard.io/"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav_link"
+              href="https://snipboard.io/"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav_link"
+              href="https://snipboard.io/"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav_link"
+              href="https://snipboard.io/e"
```

Comment:
The `href` attribute for the 'Laptops & Computers' link seems to be incorrect or incomplete (`https://snipboard.io/e`). Ensure that this URL is correct or update it to the intended destination.